website/project/presentation.rst

```diff
@@ -0,0 +1,210 @@
+in this section of the report, although it can be
+appropriate to show the code for any models you have added,
+as well as very short examples (3-4 lines) of how your code
+might be called from other components. It is also appropriate
+to link to specific parts of the ChiGame code.
+
+There is no specific length requirement for this section, but we
+expect any sufficiently-detailed section to be around 800-1200 words long.
+
+Next Steps
+~~~~~~~~~~
+
+Again, imagine that this report will be read by students who will
+continue your work in following years. Describe at least four
+tasks that those students could work on to continue the work
+you did this quarter. These can include the following:
+
+- Tasks that you started working on this quarter, but were unable
+  to finish.
+- Improvements or fixes you identified this quarter, but did
+  not work on.
+- General interesting directions that the feature could go in.
+
+Please note that you can complete this section of the report
+by simply pointing to specific GitHub issues (particularly Backlog issues)
+that future students may want to focus on. If you do so, make sure
+to include the issue number and issue title for each issue, along with
+a 2-3 sentence summary of the issue.
+
+
+Reflections
+~~~~~~~~~~~
+
+You must discuss at least three (and no more than five) aspects of the
+software development process which you found challenging and/or
+rewarding. We encourage you to decouple these as much as possible from
+the structure of this particular class. In other words, reflect upon
+what aspects of developing software as part of a team (and with other
+teams), and using the tools that facilitate this process (GitHub, Slack, etc.)
+proved to be more challenging (and/or rewarding) than
+simply working individually.
+
+You must also discuss at least one thing that you would’ve done
+differently with the benefit of hindsight.
+
+Live Demo
+---------
+
+During the final exam slot for the class, you must run a demo of your
+work meeting the following requirements:
+
+- In most cases, you must run the ChiGame website on your computer.
+  In some components (particularly the more experimental ones),
+  where there is no expectation that your code
+  will be fully integrated with ChiGame by the end of the quarter,
+  it is acceptable to show run those components by themselves (and
+  not as part of the ChiGame website)..
+- Your demo must involve running code you have written this quarter,
+  and that code *must* be in the ``dev`` branch of the repository.
+  You cannot demo code that is still in one of your branches and has
+  not gone through a code review. Note: The code doesn’t necessarily
+  have to be in the head of the ``dev`` branch. If someone breaks
+  your code in ``dev``, you can “rewind” to a previous commit in ``dev``
+  if you prefer.
+- To ensure there is enough time for all the teams to showcase their
+  work, your demo should be no more than ~10 minutes long.
```

Comment:
Given we have two hours for 6 teams, is it not okay to do 15 minutes?

Reply:
Yes, we can do 15 minutes, but if they're just going to do a demo, we'll probably tell them that they should be aiming for 10 minutes anyways.